paddle/framework/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ cc_library(ddim SRCS ddim.cc DEPS eigen3)
 cc_test(ddim_test SRCS ddim_test.cc DEPS ddim)
 nv_test(dim_test SRCS dim_test.cu DEPS ddim)
 
-cc_library(tensor SRCS tensor.cc DEPS ddim place paddle_memory device_context)
+cc_library(tensor SRCS tensor.cc DEPS ddim place paddle_memory device_context framework_proto)
```

Comment:
Tensor is decoupled from proto. Why we need the link with framework_proto?

Reply:
Serialize need the Tensor description, and serialize to a proto message is a simple way,  so link the framework_proto.